p2p/proto/block.proto

```diff
@@ -45,6 +49,8 @@ message NewBlock {
     }
 }
 
+// Requests a peer's CurrentBlockHeader
+message CurrentBlockHeaderRequest {}
```

Comment:
What does this refer to? The current/local head while syncing?

Reply:
The local head of peers to help determine what others think the `head` is. Please see this [PR](https://github.com/starknet-io/starknet-p2p-specs/pull/16)